pilot/lib/pilot/options.rb

```diff
@@ -19,6 +19,15 @@ def self.available_options
                                      description: "The bundle identifier of the app to upload or manage testers (optional)",
                                      optional: true,
                                      default_value: ENV["TESTFLIGHT_APP_IDENTITIFER"] || CredentialsManager::AppfileConfig.try_fetch_value(:app_identifier)),
+        FastlaneCore::ConfigItem.new(key: :app_platform,
```

Comment:
In my PR, I had named this `:platform` since most of the other fastlane actions use the `plugin` key as well, including deliver.